packages/venia-ui/lib/components/RadioGroup/radioGroup.js

```diff
@@ -31,15 +31,15 @@ export class RadioGroup extends Component {
     render() {
         const { classes, fieldState, items, message, ...rest } = this.props;
 
-        const options = items.map(({ label, value }) => (
+        const options = items.map(({ value, ...item }) => (
             <Radio
                 classes={{
                     label: classes.radioLabel,
                     root: classes.radio
                 }}
                 key={value}
-                label={label}
                 value={value}
+                {...item}
```

Comment:
Do we need to allow for a spread here? I feel the API is pretty limited. If you just need to allow the `disabled` prop then maybe that's all we need to add. If you want to future proof this a bit more, then spreading does accomplish the same, though it is less documentable as you can include `disabled` in the propTypes.

If you do keep this, spread `item` first and then provide the other values such as classes, key, and value.

```js
<Radio
   {...item}
   classes={...}
   key={value}
   value={value}
</Radio>
```